execution/stepExecutor.go

```diff
@@ -90,7 +91,7 @@ func (e *stepExecutor) notifyAfterStepHook(stepResult *result.StepResult) {
 
 	res := executeHook(m, stepResult, e.runner)
 	stepResult.ProtoStep.PostHookMessages = append(stepResult.ProtoStepExecResult().GetExecutionResult().Message, res.Message...)
-	stepResult.ProtoStepExecResult().GetExecutionResult().ScreenShot = append(stepResult.ProtoStepExecResult().GetExecutionResult().ScreenShot, res.ScreenShot...)
+	stepResult.ProtoStep.PostHookScreenshots = append(stepResult.ProtoStepExecResult().GetExecutionResult().ScreenShot, res.ScreenShot...)
```

Comment:
Does this break backward compatibility?
Gauge should probably retain the result in both places (`stepResult.ProtoStepExecResult().GetExecutionResult().ScreenShot` and `stepResult.ProtoStep.PostHookScreenshots`).